cmd/gnodev/repl.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	goErrors "errors"
```

Comment:
not a big fan of uppercase in import paths

Reply:
Swapped it for lowercase imports:
[14e10c2](https://github.com/gnolang/gno/pull/472/commits/14e10c27c774bcb6335525e9cdb8b9e65a583ad4)